File: Angular/IndiaTour/src/app/itinerary-pcd.ts
import { IItineraryPCD } from './iitinerary-pcd';

export class ItineraryPCD implements IItineraryPCD {
    constructor(
        public packageid: number,
        public datepackageid: number,
        public tourname: string,
        public tourimage: string,
        public description: string,
        public startdate: Date,
        public enddate: Date,
        public dayid: number,
        public daywisedescription: string,
        public location: string,
        public tourcost: number,
    ) { }
}

File: IndiaTourProject/IndiaTourProject/Models/ItineraryPCD.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace IndiaTourProject.Models
{
    public class ItineraryPCD
    {
        public int packageid { get; set; }
        public int datepackageid { get; set; }
        public string tourname { get; set; }
        public string tourimage { get; set; }
        public string description { get; set; }
        

        
        public Nullable<System.DateTime> startdate { get; set; }
        public Nullable<System.DateTime> enddate { get; set; }

        public int dayid { get; set; }
        public string daywisedescription { get; set; }
        public string location { get; set; }

        public Nullable<decimal> tourcost { get; set; }
    }
}